librz/bin/format/mach0/mach0.c

```diff
@@ -2234,10 +2222,42 @@ RzList *MACH0_(get_virtual_files)(RzBinFile *bf) {
 		vf->name = strdup(MACH0_VFILE_NAME_REBASED_STRIPPED);
 		rz_list_push(ret, vf);
 	}
+
+	// relocs
+	MACH0_(patch_relocs)
```

Comment:
Weird indentation.

Reply:
clang-format enforces this unfortunately. I can turn it off there though.